### ISSUE TITLE
False negative for `no-ignored-error`

### ISSUE THREAD

**Issue by schulzjo-tng** [DaveMBush/eslint-plugin-rxjs#131]
**Describe the bug**
One of the examples of incorrect code for `no-ignored-error` is not detected correctly:
```
source.subscribe({
  next: (value) => console.log(value)
});
```

**Repro**
Version of @smarttools/eslint-plugin-rxjs you are using: 1.0.18

Repository that has a reproduction of the problem:
The repository is private, but I can provide the following minimal reproduction (with `no-ignored-error`  set to `error`):
```
const source = {} as unknown as Observable<any>;
source.subscribe({
  next: (value) => console.log(value)
});
```
This should raise an error but does not. When I change the code to `source.subscribe((value) => console.log(value));`, the expected error appears, so my setup seems to work correctly.

**Comment by DaveMBush** [comment]
Thanks for reporting this.  version 1.0.19 should fix this issue
